<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileManage</class>
 <widget class="QDialog" name="ProfileManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Profiles</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QListWidget" name="lbx_profiles"/>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="IconButton" name="pb_new" native="true">
         <property name="text" stdset="0">
          <string>&amp;Add</string>
         </property>
         <property name="psiIconName" stdset="0">
          <string>psi/addContact</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="IconButton" name="pb_rename" native="true">
         <property name="text" stdset="0">
          <string>&amp;Rename</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="IconButton" name="pb_delete" native="true">
         <property name="text" stdset="0">
          <string>Rem&amp;ove</string>
         </property>
         <property name="psiIconName" stdset="0">
          <string>psi/remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>47</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="Line1">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QWidget</extends>
   <header>iconbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProfileManage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
